server/conf/env-var-docs.json

```diff
@@ -46,6 +46,10 @@
             "description": "URI to create a new account in the applicant identity provider.",
             "type": "string"
           },
+          "APPLICANT_PORTAL_NAME": {
+            "description": "The name of the portal that applicants log into.",
```

Comment:
Would it be helpful to have the an example of how this will be used in a sentence? eg "The name of the portal that applicants log into. Used in sentences like 'Log into your APPLICANT_PORTAL_NAME account.'"?

Reply:
Done.